common/models/zod-schemas.ts

```diff
@@ -18,3 +19,9 @@ export const createRoomSchema = z.object({
 	visibility: z.nativeEnum(Visibility).default(Visibility.Public).optional(),
 	queueMode: z.nativeEnum(QueueMode).optional(),
 });
+
+export const voteSchema = z.object({
+	service: z.enum(ALL_VIDEO_SERVICES),
+	id: z.string(),
```

Comment:
You should create a schema for `VideoId`, it gets used quite a lot and iirc you can use schemas in schemas

Reply:
Yea, I realized that when I was getting to work on the queue API endpoints yesterday because it would make the union a lot easier.